src/common/paddle/index.ts

```diff
@@ -172,6 +177,27 @@ export const isCoreTransaction = ({
   );
 };
 
+export const isOrganizationSubscription = ({
+  event,
+}: {
+  event:
+    | TransactionCreatedEvent
+    | TransactionUpdatedEvent
+    | TransactionPaidEvent
+    | TransactionCompletedEvent
+    | TransactionPaymentFailedEvent
+    | SubscriptionCreatedEvent
+    | SubscriptionUpdatedEvent
+    | SubscriptionCanceledEvent;
+}): boolean => {
+  return event.data.items.some(
+    (item) =>
+      item.price?.productId &&
+      getProductPurchaseType({ id: item.price.productId }) ===
+        ProductPurchaseType.PlusOrganization,
```

Comment:
I really wished he had a more generic approach to this 😂

Reply:
Yeah, I was thinking about it when I added it. Maybe I'll circle back later and clean that up